02_activities/assignments/assignment.sh

```diff
@@ -8,17 +8,25 @@ set -x
 # > Add your code below each comment to complete the tasks
 
 # 1. Create 5 directories named dir1, dir2, dir3, dir4, and dir5
+mkdir dir1 dir2 dir3 dir4 dir5
 
 # 2. List the contents of the current working directory to verify the presence of the 5 directories
+ls -d */
 
 # 3. Create 5 text files in dir2 named file1.txt, file2.txt, file3.txt, file4.txt, and file5.txt
+cd dir2 && touch file{1..5}.txt
 
 # 4. Append the words "Hello world" to dir2/file3.txt
+echo "Hello world" >> file3.txt
 
 # 5. Verify that file3.txt contains the words "hello world" by printing the contents of the file in the terminal
+cat file3.txt
 
 # 6. Delete file4.txt
+rm file4.txt
 
 # 7. Delete directories dir4 and dir5 including all their contents (if any)
+rm -r dir4 dir5
 
```

Comment:
remember to cd back to parent directory since you are now in dir2, but I'll mark you complete.

Reply:
Thanks Emma. I did that in the Git bash but missed copying that command in the assignment. Thank you for the suggestion. Will keep in mind in the future assignments.